tests/unit/python/foglamp/tasks/purge/test_purge.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import pytest
+import asyncio
+from unittest.mock import patch, call, MagicMock
+from foglamp.common import logger
+from foglamp.common.storage_client.storage_client import ReadingsStorageClient, StorageClient
+from foglamp.common.statistics import Statistics
+from foglamp.tasks.purge.purge import Purge
+from foglamp.common.process import FoglampProcess
+from foglamp.common.configuration_manager import ConfigurationManager
+from foglamp.common.audit_logger import AuditLogger
+
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("tasks", "purge")
+class TestPurge:
+    """Test the units of purge.py"""
+
+    def test_init(self, event_loop):
+        """Test that creating an instance of Purge calls init of FoglampProcess and creates loggers"""
+        mockStorageClient = MagicMock(spec=StorageClient)
+        mockAuditLogger = AuditLogger(mockStorageClient)
```

Comment:
One convention please either `snakecase or camelcase`. I see both naming conventions in file

Reply:
Since classes name start from Upper case, mocks of classes uses with camel case to demarcate from other mocks

Reply:
trivial